packages/maas/frontend/src/app/pages/tiers/ViewTierPage.tsx

```diff
@@ -1,10 +1,261 @@
+                                </StackItem>
+                              </Stack>
+                            }
+                          >
+                            <OutlinedQuestionCircleIcon style={{ cursor: 'pointer' }} />
+                          </Popover>
+                        </DescriptionListTerm>
+                        <DescriptionListDescription data-testid="tier-level-value">
+                          <Label isCompact>{tier?.level}</Label>
+                        </DescriptionListDescription>
+                      </DescriptionListGroup>
+                    </DescriptionList>
+                  </FlexItem>
+                </Flex>
+              </StackItem>
+              <StackItem>
+                <DescriptionList>
+                  <DetailsItem
+                    label="Description"
+                    value={tier?.description}
+                    testId="tier-description"
+                  />
+                </DescriptionList>
+              </StackItem>
+            </Stack>
+          </PageSection>
+
+          <Divider />
+
+          <PageSection>
+            <Stack hasGutter>
+              <SectionHeader
+                title="Groups"
+                description="Users in these groups will have access to this tier's models."
+              />
+              <StackItem>
+                <DescriptionList>
+                  <DetailsItem
+                    label="Groups"
+                    value={
+                      tier?.groups &&
+                      tier.groups.length > 0 && (
+                        <Stack>
+                          {tier.groups.map((group) => (
+                            <StackItem key={group}>{group}</StackItem>
+                            // Note: Originally supposed to link to the group section in the Openshift console, but we are stearing away from external links to the console.
+                          ))}
+                        </Stack>
+                      )
+                    }
+                    testId="tier-groups"
+                  />
+                </DescriptionList>
+              </StackItem>
+            </Stack>
+          </PageSection>
+
+          <Divider />
+
+          <PageSection>
+            <Stack hasGutter>
+              <SectionHeader
+                title="Models"
+                description="These models will be available to users who can access this tier."
+              />
+              <StackItem>
+                <Content className="pf-v6-u-font-weight-bold">Available models</Content>
+              </StackItem>
+              <StackItem>
+                {tier?.models && tier.models.length > 0 && (
+                  <Stack>
+                    {tier.models.map((model) => (
+                      <StackItem key={model}>{model}</StackItem>
+                    ))}
+                  </Stack>
+                )}
+              </StackItem>
+            </Stack>
+          </PageSection>
```

Comment:
You should be able to remove the models part of this since we won't have it for MVP